ee/ucp/deploy-application-package.md

```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
 
 6. Select either **Compose File** or **App Package** for the **Application File Mode**.
 7. Select **Next**.
-8. If you selected Compose file, enter or upload your `docker-compose.yml` in **2. Add Application File**.
+8. If you selected Compose file, enter or upload your `docker-compose.yml` in the **Create Application** window.
```

Comment:
Change to read: Add the pertinent application file, according to the selected Application File Mode.